test-prow-e2e.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+set -eExuo pipefail
+
+NAMESPACE=""
+DF_CONSOLE_IMAGE=""
```

Comment:
data federation namespace is finalized, right ? we can use that here now I think...
```suggestion
NAMESPACE=""
DF_CONSOLE_IMAGE="$1"
```

Reply:
The `df-console-image` will replace the "" that's there right now instead of us passing that in as an arg.